test/bats/akeyless.bats

```diff
@@ -0,0 +1,91 @@
+#!/usr/bin/env bats
+
+ load helpers
+
+ BATS_TESTS_DIR=test/bats/tests/akeyless
+ WAIT_TIME=120
+ SLEEP_TIME=1
+
+
+ export LABEL_VALUE=${LABEL_VALUE:-"test"}
+ export ANNOTATION_VALUE=${ANNOTATION_VALUE:-"app=test"}
+
+@test "install akeyless csi provider" {
+   # create the ns akeyless
+   kubectl create ns akeyless
+   # install the akeyless provider using the helm charts
+
+   helm repo add akeyless https://akeylesslabs.github.io/helm-charts
+   helm repo update
+   helm install akeyless akeyless/akeyless-csi-provider --namespace=akeyless --set csi.image.pullPolicy=Always
+
+   # wait for akeyless and akeyless-csi-provider pods to be running
+   kubectl wait --for=condition=Ready --timeout=${WAIT_TIME}s pods --all -n akeyless
+ }
+
+@test "create akeyless k8s secret" {
+  run kubectl create secret generic secrets-store-creds --from-literal akeylessAccessKey=${AKEYLESS_ACCESS_KEY}
```

Comment:
Maybe check if `AKEYLESS_ACCESS_KEY` is defined similar to https://github.com/kubernetes-sigs/secrets-store-csi-driver/blob/2beb3ecc8d500daac4f8574e0c7b7867e1519583/test/bats/azure.bats#L43-L48 so the tests can fail early when required creds aren't available?

Reply:
fixed